packages/editor-ui/src/mixins/executionsHelpers.ts

```diff
@@ -50,34 +50,26 @@ export const executionHelpers = mixins(genericHelpers).extend({
 			) {
 				status.name = 'running';
 				status.label = this.$locale.baseText('executionsList.running');
-				if (execution.startedAt) {
-					status.runningTime = this.displayTimer(
-						new Date().getTime() - new Date(execution.startedAt).getTime(),
-						true,
-					);
-				}
 			} else if (execution.status === 'success' || execution.finished) {
 				status.name = 'success';
 				status.label = this.$locale.baseText('executionsList.succeeded');
-				if (execution.stoppedAt) {
-					status.runningTime = this.displayTimer(
-						new Date(execution.stoppedAt).getTime() - new Date(execution.startedAt).getTime(),
-						true,
-					);
-				}
-			} else if (
-				execution.status === 'failed' ||
-				execution.status === 'crashed' ||
-				execution.stoppedAt !== null
-			) {
+			} else if (execution.status === 'failed' || execution.status === 'crashed') {
 				status.name = 'error';
 				status.label = this.$locale.baseText('executionsList.error');
-				if (execution.stoppedAt) {
-					status.runningTime = this.displayTimer(
-						new Date(execution.stoppedAt).getTime() - new Date(execution.startedAt).getTime(),
-						true,
-					);
-				}
+			} else if (execution.status === 'canceled') {
+				status.label = this.$locale.baseText('executionsList.canceled');
+			}
+
+			if (!execution.status) execution.status = 'unknown';
+
+			if (execution.startedAt) {
```

Comment:
Agree. I think we should just not be showing timer is there is no `stoppedAt`, so that should be:
```
if (execution.startedAt && execution.stoppedAt) {
```
And [here](https://github.com/n8n-io/n8n/blob/221645576087e4cd828b34ea33e874e1bff5f34a/packages/editor-ui/src/components/ExecutionsView/ExecutionCard.vue#L40) we need:
```
<n8n-text
v-else-if="
	executionUIDetails.name !== 'waiting' &&
	executionUIDetails.name !== 'unknown' &&
	executionUIDetails.runningTime !== ''
"
:color="isActive ? 'text-dark' : 'text-base'"
size="small"
>
{{
	$locale.baseText('executionDetails.runningTimeFinished', {
		interpolate: { time: executionUIDetails?.runningTime },
	})
}}
</n8n-text>
```